Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.21 AS builder
```

Comment:
since you're doing this, you may want to go to ubi9 and go 1.22 https://catalog.redhat.com/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690. At least I would be consistent in using ubi9 9.5 all over the place.